### ISSUE TITLE
Back button or gesture should close comments, not the entire app

### ISSUE THREAD

**Issue by eriktews** [pixelfed/pixelfed-rn#98]
When you open the comments on Android and then use the "back" gesture (that's swiping from the edge of the screen to the center on the Pixel 7) then the entire app is closed. Instead I would expect that just the comments should be closed.

**Comment by Kwasow** [comment]
Yes! This is indeed the same issue. Closing this one as completed
